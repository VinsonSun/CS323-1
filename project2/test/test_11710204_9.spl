int add(int x, int y, int z)
{
    return x + y + z;
}
int test_11710204_9()
{
  int a = 2;
  int b = 3;
  int c = add(a,b);
  return c;
}
