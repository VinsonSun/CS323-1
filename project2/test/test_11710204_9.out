Error type 9 at Line 9: invalid argument type for add
